README.md

```diff
@@ -1 +1,343 @@
+# error does inherit from ChefCore::Error, argument handling will be done automatically.
+
+class MyError < ChefCore::Error
+  def initialize()
+    super("BADFILE001", "exe")
+  end
+end
+
+ChefCore::ErrorPrinter.show_error(MyError.new, {stack_trace_path: '/my/stack', log_location: 'my/log'})`
+```
+
+Output given the dispaly defaults and the definition above:
+
+```
+BADFILE001
+
+File extension 'exe' is unsupported. Use '.rb' instead.
+
+If you are not able to resolve this issue, please contact Chef support
+at workstation@chef.io and include the log file and stack trace from the locations
+below:
+
+  /my/stack
+  my/log
+```
+
+As seen in the example, BADFILE001 takes a parameter. Your exception will need to provide a member 'parameters'
+that returns a list of error message parameters. Exceptions derived from `ChefCore::Error` will take
+care of this for you.  You may also note references to Chef support and emails - this is the default
+message footer, which is described in the next section.
+
+
+### Overriding Common Elements
+
+Several common elements are defined in `chef-core/i18n/errors/en.yml`. It is likely that you'll want to
+override these in your own `errors/LANG.yml`:
+
+* `footer` - contains footers to show when decorations = true, footer = true.  Content
+             will vary based on whether `stack` and `log` are true.  Contains separate subkeys:
+  * `both` - footer to dispaly when both stack and log are true
+  * `log_only` - text when only log is true
+  * `stack_only` - text when only stack is true
+  * `neither` - text when neither one is true
+* `header` - display the message header, which is the error ID.
+
+
+## Actions
+
+The gem `chef-core-actions` (same repository) includes pre-defined actions that can be run such as installing Chef
+client on a remote system, or converging a remote system; and a light framework for defining
+additional actions. Our intent with this component is to build a library of reusable actions
+that are shared across Chef tooling and beyond.
+
+Actions evolved out of some of our early chef-run work, when we were looking to provide
+multiple new tools with common functionality. They were developed to provide self-contained
+actions that have no direct interface to the user/terminal or any external configuration providers.
+This makes them well-suited for use in any existing CLI tool, even if that tool is not making
+use of Chef Core.
+
+This framework is minimal - it defines a base class, and invokes provided overrides/callbacks.
+Invoking the actions is left to the calling application.
+A well-written, shareable Action...
+
+* ...informs the listener of what it's doing via :notify, so that the listener can pass it along to the operator
+in whatever way is appropriate for the application.
+* ...does not perform any user-facing actions (such as requesting input or dispalying results)
+* ...has no knowledge of configuration options loaded from an external system. Instead it will
```

Comment:
This sentence - "Instead it will" - just ends and does not make sense when concatenated to the next line. Is there a sentence missing here?

Reply:
Leftover from a prior iteration. Will remove.